classes/class.memberorder.php

```diff
@@ -1,6 +1,407 @@
 <?php
 	class MemberOrder
 	{	
+		/**
+		 * The Member Order ID
+		 *
+		 * @since TBD
+		 *
+		 * @var int
+		 */
+		public $id = 0;
```

Comment:
What is the reason for using public class member variables, and not using private with __get(), __set() magic methods, and adding public get(), set() methods to allow transitioning to a more expected class behavior?

Reply:
Hi @eighty20results, thank you for your feedback. As I mentioned in my review of this PR, transitioning to magic getters and setters is something that we are going to look into after the other notes are addressed. Having public properties seemed like the "quick solution" to stop using dynamic properties ([being deprecated in PHP 8.2](https://php.watch/rfcs/deprecate_dynamic_properties)), but it does seem like there is a cleaner way that we can do this. We will update this PR as decisions are made on this.

Reply:
Thanks for sharing this @eighty20results